test/core.vwtest.json

```diff
@@ -5013,5 +5013,17 @@
     "input_files": [
       "train-sets/regression_simple_1var_overflow.txt"
     ]
+  },
+  {
+    "id": 389,
+    "desc": "Test using automl with ccb",
+    "vw_command": "--ccb_explore_adf --example_queue_limit 7 -d train-sets/ccb_reuse_small.data --automl 3",
+    "diff_files": {
+      "stderr": "train-sets/ref/ccb_reuse_small_automl.stderr",
```

Comment:
How is this test confirming the change? Shouldn't verbose extra metrics be included into references?